src/pages/whats-new.js

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { Container, Row, Col, ListGroup } from 'react-bootstrap';
+import { Link, Element } from "react-scroll";
+
+import twitterIcon from '../images/icons/twitter-icon-blue.svg'
+import NewFeature from "../components/new-feature"
+import Layout from "../components/layout"
+import SEO from "../components/seo"
+
+import newFeaturesData from "../../data/new-features.json"
+import image from "../images/feature-june.gif"
+
+import "../styles/pages/whats-new.scss"
+
+const dateToDivID = (date) => {return "#" + date.replace(/[\W_]+/g, '-').toLowerCase()}
+
+const WhatsNew = () => (
+  <Layout pageClassName="whats-new-page">
+    <SEO title="What's New" />
+    <Container>
+    {/*<Row><Col><h2>What's New</h2></Col></Row>*/}
```

Comment:
If this looks OK, could we put it in to make it obvious what the content is? Or if you can think of a better way...

Reply:
Hi John, is putting the title back be better?